### DISCUSSION TITLE
Welcome to eZ_Navi-Delete Discussions!

### DISCUSSION THREAD

**Discussion by radensb** [radensb/eZ_Navi-Delete#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by slides3305** [comment]
Hi there!  Just saw a bug fix 30 minutes ago, so I figured I would say hi.  '09 370z Nav owner here.  Can't wait to get my hands on one of your boards, hopefully you get a batch in soon, I'm ready to purchase on Tindie if so, thanks for all the great work you have done!

**Reply by radensb** [reply]
Hey Slides,

Thanks for the interest in the project! It has been a lot of work, but also very fun. I am waiting on some parts that I want to include on the new boards to test on an existing board. Then the design will be ready to send out to build a new batch. Ill post on Tindie and all that have joined the wait list will get a notification. My hope is that there will be no issues, but with the global chip shortage and the fact that the eZ Navi Delete uses automotive parts, the lead times could be affected. I should know soon.

**Comment by Ghostvette** [comment]
Raden, I built a set of cables for Leingod on the the 370Z forum. Good job on the diagrams and the links to suppliers.

**Comment by Davesepicrage** [comment]
Hi raden, thanks again for this amazing product. Just a couple questions, I see that there is resistive controls that can be retained via your PCB however im unfamiliar with the wiring of the Cruise control module, would this install effect this? or is it untouched by the install?

Thanks!

**Reply by radensb** [reply]
Just to be clear, the cruse controls are a separate thing, and not related to the stereo buttons on the other side.

The options depend on the install you are planning and what is currently implemented in the firmware. Many aftermarket stereos use the resistive button interface so all that needs to be done is to wire the signals from the stereo connector to the new head unit and map the buttons to the functions you want. The eZ Navi-Delete board _does_ have the interface for those button controls. The intent was thattheir state can be sent to a system (most likely a tablet running software to interpret them) to control it. However, that requires firmware and app implementations that have not been done yet. It is something that can be easily added later, so it wouldn't affect an install.

**Comment by XiJingPooh** [comment]
Will this work with non-navi systems?

**Reply by radensb** [reply]
No, it wouldn't as the non-navi Z does not control the AC system over the CAN Bus. The non-navi models wouldn't need the translation interface this provides anyway because it already uses the three dial control module, corresponding AC amp, and is compatible with all existing aftermarket stereo equipment and bezels.

Is there a specific aspect this board provides that you're interested in?

**Reply by XiJingPooh** [reply]
Was trying to see if I could use this to use steering wheel controls and display ac info for a braniac install.

**Reply by radensb** [reply]
Ah, ok. I don't see any reason why you couldn't use this platform to relay the steering wheel controls to a tablet (although, that work is planned, it has not been started yet). If there was firmware in place on the eZ Navi-Delete, you could wire the two steering wheel control signals to the Steering Wheel Control connector and send a serial packet from the USB to a tablet. That was the intended purpose of that port, I just haven't had time to work it yet. The other side of that is that there needs to be an app running on the Brainiac system that listens for those packets and controls the appropriate Android feature (volume, track navigation, etc). The platform I was targeting for this was [CarDroid](https://github.com/rampage128/cardroid) but I haven't heard from them in a while.

As for the AC status, it sounds like you want to redundantly display the state of the AC system on a screen that the dial module is indicating. Unfortunately, this wouldn't allow for that. The eZ Navi-Delete works by translating the protocol of the base model AC dial control module to the CAN bus and translating the CAN status back to the dial module to update things like the LED's. The base model Z's don't have AC CAN control and thus don't provide this capability. What you would need to do is make something that sniffs the dial module Rx line and relays the AC state data to a tablet app that displays it. A 3.3V Arduino with a simple interface circuit could do that, but there is a lot of other pieces missing at this point.

Does Brainiac not have a steering wheel control solution of their own yet?

**Reply by XiJingPooh** [reply]
Thanks. I opted not to get their mk2 electronic package since I've read less than pleasant reviews and the static feedback ppl were having.

I found a post and YouTube video where a Korean company makes this module to convert the steering wheel switch input into keyboard inputs that the tablet will recognize.  Tried it with a keyboard and was able to control a bunch of things so I picked one up.  Also planning on adding a video input for a reverse camera and digital to analog converter to the amp.
